app/src/main/java/pl/ludek/smat/home/pool_office_client/domain/usecases/OffRelayFirst.kt

```diff
@@ -2,17 +2,18 @@ package pl.ludek.smat.home.pool_office_client.domain.usecases
 
 import android.util.Log
 import androidx.lifecycle.MutableLiveData
-import pl.ludek.smat.home.pool_office_client.data.apiservice.ApiModule
+import pl.ludek.smat.home.pool_office_client.data.apiservice.PoolInfoClient
 import pl.ludek.smat.home.pool_office_client.domain.model.RelayData
 import retrofit2.Call
 import retrofit2.Callback
 import retrofit2.Response
 
 class OffRelayFirst: SingleUseCaseButton {
-    private var client = ApiModule.retrofitService
+    private var client = PoolInfoClient
```

Comment:
This might look weird but is totally fine, in the future we can use DI (Dependency Injection) to inject the client.